recipes/filepattern/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "filepattern" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }} 
+
+source:
+  url: https://github.com/PolusAI/filepattern/archive/v{{ version }}.tar.gz
+  sha256: d5129345fc46bba18716757ea64b3783ce1a033aa81a437f5030a30bcdc670a7
+
+build:
+  skip: true  # [py2k]
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - make  # [not win]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - git
+
+outputs:
+  - name: filepattern
+    script: build.sh  # [unix]
+    script: bld.bat  # [win]
+    version: {{ version }}
+    requirements:
+      build:
+        - cmake
+        - make  # [not win]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - git
+      host:
+        - python
+        - setuptools
+        - pip
+        - pybind11
+      run:
+        - python
+        - pydantic
+
+    test:
+      imports:
+        - filepattern
+      commands:
+        - pip check
+      requires:
+        - pip
+        - pytest
```

Comment:
These tests are not being run, and the python packages are empty. Maybe the syntax is incorrect for this outputs section? It might be because this output shares the same name as the top level package.